test/system/src/test/java/io/pravega/test/system/MultiReaderWriterWithFailOverTest.java

```diff
@@ -289,21 +287,21 @@ public void multiReaderWriterWithFailOverTest() throws Exception {
             //delete readergroup
             log.info("Deleting readergroup {}", readerGroupName);
             readerGroupManager.deleteReaderGroup(readerGroupName);
+            //seal the stream
```

Comment:
Don't we have to make the same change in `MultiReaderTxnWriterWithFailoverTest `?